pkg/kubectl/describe.go

```diff
@@ -545,7 +545,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 			}
 		}
 		describeVolumes(pod.Spec.Volumes, out, "")
-		fmt.Fprintf(out, "QoS Class:\t%s\n", qos.GetPodQOS(pod))
+		fmt.Fprintf(out, "QoS Tier:\t%s\n", qos.GetPodQOS(pod))
```

Comment:
Sorry, this isn't obvious to me.  This looks like the wording change was intentional here: https://github.com/kubernetes/kubernetes/pull/27749 and we don't lock the shape of describers as API.  Why does this need to change back?